src/UserService.Models.Db/DbUserEducation.cs

```diff
@@ -1,52 +1,63 @@
 ﻿using Microsoft.EntityFrameworkCore;
 using Microsoft.EntityFrameworkCore.Metadata.Builders;
 using System;
+using System.Collections.Generic;
 
 namespace LT.DigitalOffice.UserService.Models.Db
 {
-    public class DbUserEducation
+  public class DbUserEducation
+  {
+    public const string TableName = "UserEducations";
+
+    public Guid Id { get; set; }
+    public Guid UserId { get; set; }
+    public string UniversityName { get; set; }
+    public string QualificationName { get; set; }
+    public int FormEducation { get; set; }
+    public DateTime AdmissionAt { get; set; }
+    public DateTime? IssueAt { get; set; }
+    public bool IsActive { get; set; }
+    public Guid CreatedBy { get; set; }
+    public DateTime CreatedAtUtc { get; set; }
+    public Guid? ModifiedBy { get; set; }
+    public DateTime? ModifiedAtUtc { get; set; }
+    public DbUser User { get; set; }
+
+    public ICollection<DbUserEducationImage> Images { get; set; }
+
+    public DbUserEducation()
     {
-        public const string TableName = "UserEducations";
-
-        public Guid Id { get; set; }
-        public Guid UserId { get; set; }
-        public string UniversityName { get; set; }
-        public string QualificationName { get; set; }
-        public int FormEducation { get; set; }
-        public DateTime AdmissionAt { get; set; }
-        public DateTime? IssueAt { get; set; }
-        public bool IsActive { get; set; }
-        public Guid CreatedBy { get; set; }
-        public DateTime CreatedAtUtc { get; set; }
-        public Guid? ModifiedBy { get; set; }
-        public DateTime? ModifiedAtUtc { get; set; }
-
-        public DbUser User { get; set; }
+      Images = new List<DbUserEducationImage>();
```

Comment:
List? mb hashSet?

Reply:
i dont this that its better. i don think that we can have over 10 images for one education. List is enought for few entities.